index.Rmd

```diff
@@ -14,29 +14,60 @@ library(rgdal)
 library(broom)
 ```
 
-# Preface {-}
+# Preface and motivation {-}
```

Comment:
I want us to add a section here for *R is open source* and the different expectations that you have to have with that. R puts you at the cutting edge of techniques but its constantly changing. The more you learn you will need to begin managing the environments you are working in or could get caught out with the old *it worked last year* problem.